### DISCUSSION TITLE
How to generate structures with Z'=2?

### DISCUSSION THREAD

**Discussion by wind-show** [Tomoki-YAMASHITA/CrySPY#76] [Q&A]
The default setting is generating structures with Z'=1.Is it possible to generate a Z '=2 structure?Thanks for the help in advance!

**Comment by Tomoki-YAMASHITA** [comment]
You want to use conventional cells (Z = 1, 2, or 4)?
Currently, switching between primitive and conventional structures cannot be done from the input file.
We plan to include a switching option in the next version.


If you want to use conventional cells in the current version, locate the installed gen_pyxtal.py (CrySPY/src/cryspy/RS/gen_struc_RS/gen_pyxtal.py) file and modify it as follows:
```
conventional=False
```
↓↓↓
```
conventional=True
```
There are four locations within the file.

**Reply by wind-show** [reply]
Thank you for your patience and thoughtful answer. I'm sorry I didn't make the question clear. Z' is the number of molecules in an asymmetric unit. When Z' ≠ 1, it presents basic building block is not just one molecule, more or less. And there are structures which are Z' ≠ 1. It seems CrySPY has a default setting of Z' = 1. The ability to set this parameter is not found in the description of the input file.Is that possible to change the Z'?

**Reply by Tomoki-YAMASHITA** [reply]
No, we don't use Z'.

**Reply by wind-show** [reply]
Okay, I get it. Thank you!
